docs/index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html class="scroll-smooth" lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
A crossorigin reference to anonymous can be added to the script tag

Reply:
https://github.com/tailwindlabs/tailwindcss.com/issues/441

Seems like it's not supported.
You can ignore this.